modules/base/package.json

```diff
@@ -28,6 +28,7 @@
     "aesthetic": "^5.1.0",
     "aesthetic-adapter-aphrodite": "^5.1.0",
```

Comment:
should we then remove this one?

Reply:
but we should keep aware of this for future updates, mainly when we do update aesthtetic

Reply:
Ok, will do!